src/store/sqlite/mod.rs

```diff
@@ -151,6 +177,56 @@ pub enum ValueError {
     StoredType(u8),
 }
 
+/// Dimension of the database
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Deserialize, Serialize)]
+#[serde(tag = "unit")]
+pub enum Size {
+    /// Dimension expressed in KiloBytes
+    #[serde(rename = "kb")]
+    Kb(NonZeroU64),
+    /// Dimension expressed in MegaBytes
+    #[serde(rename = "mb")]
+    Mb(NonZeroU64),
+    /// Dimension expressed in GigaBytes
+    #[serde(rename = "gb")]
+    GB(NonZeroU64),
+    /// Dimension expressed in KibiBytes
+    #[serde(rename = "kib")]
+    KiB(NonZeroU64),
+    /// Dimension expressed in MebiBytes
+    #[serde(rename = "mib")]
+    MiB(NonZeroU64),
+    /// Dimension expressed in GibiBytes
+    #[serde(rename = "gib")]
+    GiB(NonZeroU64),
+}
+
+impl Size {
+    /// Convert the size to bytes
+    const fn to_bytes(self) -> u64 {
+        match self {
+            Size::Kb(kb) => kb.get().saturating_mul(1000),
+            Size::Mb(mb) => mb.get().saturating_mul(1000 * 1000),
+            Size::GB(gb) => gb.get().saturating_mul(1000 * 1000 * 1000),
+            Size::KiB(kib) => kib.get().saturating_mul(1024),
+            Size::MiB(mib) => mib.get().saturating_mul(1024 * 1024),
+            Size::GiB(gib) => gib.get().saturating_mul(1024 * 1024 * 1024),
```

Comment:
Add a test for each unit to check the conversion to_bytes and to_kibi

Ex. `assert_eq!(Size::Kb(1), 1000)`